tox.ini

```diff
@@ -0,0 +1,45 @@
+[tox]
+envlist = py{27,37}, docs
+skip_missing_interpreters = true
+
+
+[testenv]
+passenv = SSL_CERT_FILE CI TRAVIS* APPVEYOR* CODECOV*
+setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
+usedevelop = True
+deps = -rrequirements.txt
+       -rtest/requirements.txt
+commands = pytest {posargs:-vv --cov-report term-missing --cov=sureal/}
+
+[testenv:coverage]
+passenv = {[testenv]passenv}
```

Comment:
Test code coverage is very handy, we can even upload that coverage to services like codecov (https://codecov.io/).
Same as travis (free for open source projects)